pickfirst.go

```diff
@@ -168,7 +173,12 @@ func (b *pickfirstBalancer) UpdateClientConnState(state balancer.ClientConnState
 	return nil
 }
 
+// UpdateSubConnState is unused as a StateListener is always registered when
+// creating SubConns.
 func (b *pickfirstBalancer) UpdateSubConnState(subConn balancer.SubConn, state balancer.SubConnState) {
+}
```

Comment:
Should we throw an error log here, just to ensure that it is caught by our tests *if* by any chance this ends up happening?

Reply:
Good idea! Done.

Reply:
LGTM